superset/db_engine_specs/pinot.py

```diff
@@ -99,12 +101,13 @@ def get_timestamp_expr(
         else:
             seconds_or_ms = "MILLISECONDS" if pdf == "epoch_ms" else "SECONDS"
             tf = f"1:{seconds_or_ms}:EPOCH"
+
         if time_grain:
             granularity = cls.get_time_grain_expressions().get(time_grain)
             if not granularity:
                 raise NotImplementedError(f"No pinot grain spec for '{time_grain}'")
         else:
-            return TimestampExpression("{{col}}", col)
+            return TimestampExpression("{col}", col)
```

Comment:
Would you also mind adding some unit tests for Pinot which cover the `get_timestamp_expr` function. You can find many other examples of this in the other DB engine specs.

Reply:
Sure, not a problem.